events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,31 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { TicketModel } from "../tickets/tickets.model";
+import { EventCreateDto } from "./dto/requests/events-create.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async createEvent(newEvent: EventCreateDto): Promise<EventModel[]> {
+    return await EventsRepository.create(newEvent);
+  }
+
+  static async updateEvent(
+    event: EventCreateDto,
+    id: string
+  ): Promise<null | EventModel[]> {
+    return await EventsRepository.update(event, id);
```

Comment:
Here you returning an array of records, please return a single record

Reply:
done 
![image](https://github.com/user-attachments/assets/55e46728-5ce7-4310-8ac7-7e3d37cca9be)